### PR TITLE
Add seed input to Ollamavision

### PR DESCRIPTION
**Pull request by vilanele**
Add seed input to ollamavision.

### CODE REVIEW

**Comment by stavsap** [comment]
please update version here https://github.com/stavsap/comfyui-ollama/blob/main/pyproject.toml

File: CompfyuiOllama.py [hunk 1]
```diff
@@ -39,6 +40,7 @@ def INPUT_TYPES(s):
                     "default": "http://127.0.0.1:11434"
                 }),
                 "model": ((), {}),
+                "seed": ("INT", {"default": seed, "min": 0, "max": 2 ** 31, "step": 1}),
```
Comment (stavsap) [inline CompfyuiOllama.py:43]: please put the seed value in the end of the list to not break existing workflows.

Comment (vilanele) [reply CompfyuiOllama.py]: Done.

**Comment by vilanele** [comment]
All missing changes added in new commit.

**Pull request merged**